diagnostic.md

```diff
@@ -7,6 +7,8 @@ Record your responses inside the fenced code blocks below each question.
 
     ```md
     <!-- your response here -->
+    Inside the Ember Application Router you are telling the app what view each route is referring to,
+    inside the ember route we build a model hook for our data.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.